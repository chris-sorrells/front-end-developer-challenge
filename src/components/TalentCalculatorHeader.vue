<template>
  <div class="talent-calculator-header">
    <slot />
  </div>
</template>

<style lang="scss">
.talent-calculator-header {
  display: block;
  margin-bottom: 50px;
  padding: 10px;
  font-weight: bold;
  text-align: center;
  font-size: 130%;
  border: 1px solid rgba(255, 255, 255, 0.15);
  background-color: rgba(128, 128, 128, 0.25);

  @media (max-width: 950px) {
    margin-bottom: 25px;
  }
}
</style>
