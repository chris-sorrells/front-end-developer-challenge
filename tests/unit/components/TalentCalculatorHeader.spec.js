import { shallowMount } from "@vue/test-utils";
import TalentCalculatorHeader from "@/components/TalentCalculatorHeader.vue";

describe("TalentCalculatorHeader.vue", () => {
  it("renders", () => {
    const wrapper = shallowMount(TalentCalculatorHeader);
    expect(wrapper.element).toMatchSnapshot();
  });
});
