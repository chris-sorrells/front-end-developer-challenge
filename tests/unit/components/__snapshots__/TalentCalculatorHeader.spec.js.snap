// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TalentCalculatorHeader.vue renders 1`] = `
<div
  class="talent-calculator-header"
/>
`;
